lnrpc/routerrpc/router.proto

```diff
@@ -430,7 +422,7 @@ service Router {
     differs from SendPayment in that it allows users to specify a full route
     manually. This can be used for things like rebalancing, and atomic swaps.
     */
-    rpc SendToRoute (SendToRouteRequest) returns (SendToRouteResponse);
+    rpc SendToRoute (SendToRouteRequest) returns (lnrpc.HTLCAttempt);
```

Comment:
don't we need to make this `SendToRouteV2` for backcompat?

Reply:
Yes, added